apertium-tur-uzb.tur-uzb.dix

```diff
@@ -3641,8 +3652,9 @@
     <e><p><l>oyun<s n="n"/></l><r>oʻyin<s n="n"/></r></p></e>
     <e><p><l>kuş<s n="n"/></l><r>parranda<s n="n"/></r></p></e>
     <e><p><l>kuşlar<s n="n"/></l><r>parranda<s n="n"/></r></p></e>
+    <e><p><l>alt<s n="n"/></l><r>tag<s n="n"/></r></p></e> <!--This translation(tag) weighs more than "ost" IMHO -->
```

Comment:
You can set a default by either putting in an lrx rule (if you sometimes want the less common one), or by limiting the translation of the one you think is worse to only go one direction (if you never want to translate to the less common one, but still want to translate from it).

Reply:
You can also put a weight in the bidix, e.g. `<e w="2.0">`